test/snapshots/blueprint.test.ts.md

```diff
@@ -11,6 +11,7 @@ Generated by [AVA](https://avajs.dev).
     {
       resources: {
         foo: {
+          description: '',
```

Comment:
Can you add a description to one of the fixture resources (foo)?

Reply:
311d4df8bb0ce9a2d2d7b52b488269f502e30da9